web/src/lib/domain-logic/essentialQuestions.ts

```diff
@@ -0,0 +1,155 @@
+import { EssentialQuestionsProps, ProfileLabelField } from "@/lib/types/types";
+import {
+  emptyIndustrySpecificData,
+  Industry,
+  IndustrySpecificData,
+  LookupIndustryById,
+  ProfileData,
+} from "@businessnjgovnavigator/shared";
+
+type EssentialQuestionFunction = (industry: Industry) => boolean;
```

Comment:
this type name is super generic and I think it should be renamed if we're only using it as a validation function type?